ush/setup.py

```diff
@@ -568,52 +573,154 @@ def _remove_tag(tasks, tag):
     #
     # -----------------------------------------------------------------------
     #
-    # Remove all verification [meta]tasks for which no fields are specified.
+    # Set some variables needed for running checks on and creating new
+    # (derived) configuration variables for the verification.
+    #
+    # -----------------------------------------------------------------------
+    #
+    date_first_cycl = workflow_config.get("DATE_FIRST_CYCL")
+    date_last_cycl = workflow_config.get("DATE_LAST_CYCL")
+    incr_cycl_freq = int(workflow_config.get("INCR_CYCL_FREQ"))
+    fcst_len_hrs = workflow_config.get("FCST_LEN_HRS")
+
+    # Set the forecast output interval.  Ideally, this should be obtained
+    # from the SRW App's configuration file, but such a variable doesn't
+    # yet exist in that file.
+    fcst_output_intvl_hrs = 1
+    workflow_config['FCST_OUTPUT_INTVL_HRS'] = fcst_output_intvl_hrs
```

Comment:
Since it's safe to assume that this value will be used in the future, I say you should add it to config_defaults, even if the only valid value is 1 for now. It will make things easier for the future hypothetical person who adds that capability.